composer.json

```diff
@@ -22,7 +22,8 @@
     "zendframework/zend-diactoros": "~1.1",
```

Comment:
Yeah... But I think we can assume PHP7 for now as I'm unsure about when Apigility 2 will be released. So let's assume PHP7+ for now :).